dependabot-core.gemspec

```diff
@@ -32,8 +32,12 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "aws-sdk-ecr", "~> 1.5"
   spec.add_dependency "bundler", ">= 1.16", "< 3.0.0"
+  spec.add_dependency "cocoapods", "~> 1.5.3"
```

Comment:
We won't be able to merge this whilst this dependency pins `activesupport` to `4.2.10`. Hopefully we can solve that, though.